### ISSUE TITLE
Webpack not watching bundle.js

### ISSUE THREAD

**Issue by jnschrag** [ixkaito/frasco#33]
Hi there! First, I'd like to say thank you for the wonderful project. It's one of the better boilerplate Jekyll setups that I've found, for sure.

I've come across a few issues related to the Webpack/Gulp configuration.

1. Trying to use ES6 code results in a compiling error. I was able to fix this by changing the order of the Gulp pipeline in `webpack.js` to putting Uglify immediately after Babel. Code for that is below
```
gulp.task('webpack', function () {
  return gulp.src(entry)
    .pipe(plumber())
    .pipe(named())
    .pipe(babel())
    .pipe(uglify())
    .pipe(webpackStream(config.webpack, webpack))
    .pipe(gulp.dest(config.assets + '/' + config.js.dest));
});
```

2. This is a bigger issue that I haven't been able to resolve, but Webpack doesn't seem to be rebundling `bundle.js` when you make changes to it. The only way to force it to recompile is by restarting the `start` task. Has anybody else experienced this problem/found a solution? It makes doing any JavaScript development incredibly tedious.

Thanks!

**Comment by ixkaito** [comment]
@jnschrag Thanks for your issue.

It seems that the npm packages update stops the JavaScript compiling. I haven't taken a closer look at each of packages, so I don't know which package is the cause.

Just reverted the last two commits. Please pull Frasco again and make sure to remove the `node_modules` folder before re-install npm packages by `npm i`.

**Comment by jnschrag** [comment]
Thanks for looking into this, it's much appreciated!

**Comment by ixkaito** [comment]
@jnschrag Released v1.0.0. Please have a try 😃
